src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsViewModel.java

```diff
@@ -0,0 +1,342 @@
+            AbbreviationViewModel abbViewModel = new AbbreviationViewModel(abbreviation);
+            if (abbreviations.contains(abbViewModel)) {
+                if (abbViewModel.equals(currentAbbreviation.get())) {
+                    setCurrentAbbreviationNameAndAbbreviationIfValid();
+                } else {
+                    throw new JabRefException("Duplicated journal abbreviation");
+                }
+            } else {
+                setCurrentAbbreviationNameAndAbbreviationIfValid();
+            }
+        }
+    }
+
+    /**
+     * Sets the name and the abbreviation of the {@code currentAbbreviation} property
+     * to the values of the {@code abbreviationsName} and {@code abbreviationsAbbreviation}
+     * properties. If the values are not valid for the abbreviation it will throw a {@link JabRefException}.
+     *
+     * @throws JabRefException
+     */
+    private void setCurrentAbbreviationNameAndAbbreviationIfValid() throws JabRefException {
+        String name = abbreviationsName.get();
+        String abbreviation = abbreviationsAbbreviation.get();
+        if (name.trim().isEmpty()) {
+            throw new JabRefException("Name can not be empty");
+        } else if (abbreviation.trim().isEmpty()) {
+            throw new JabRefException("Abbrevaition can not be empty");
+        } else {
+            currentAbbreviation.get().setName(name);
+            currentAbbreviation.get().setAbbreviation(abbreviation);
+        }
+    }
+
+    /**
+     * Method to delete the abbreviation set in the currentAbbreviation property.
+     * The currentAbbreviationProperty will be set to the previous or next abbreviation
+     * in the abbreviations property if applicable. Else it will be set null.
+     */
+    public void deleteAbbreviation() {
+        int index = abbreviations.indexOf(currentAbbreviation.get());
+        // index > 1 because there is a pseudo abbreviation at the start of every editable abbreviation list
+        if (index > 1) {
+            currentAbbreviation.set(abbreviations.get(index - 1));
+        } else if ((index + 1) < abbreviationsCount.get()) {
+            currentAbbreviation.set(abbreviations.get(index + 1));
+        } else {
+            currentAbbreviation.set(null);
+        }
+        abbreviations.remove(index);
+    }
+
+    /**
+     * Calls the {@link AbbreviationsFile#WriteOrCreate()} method for each file
+     * in the journalFiles property which will overwrite the existing files with
+     * the content of the abbreviations property of the AbbriviationsFile. Non
+     * existing files will be created.
+     */
+    public void saveJournalAbbreviationFiles() {
+        journalFiles.forEach(file -> {
+            try {
+                file.WriteOrCreate();
+            } catch (FileNotFoundException e) {
+                logger.debug(e.getLocalizedMessage());
+            } catch (IOException ioe) {
```

Comment:
I found out `FileNotFoundException` is even redundant if I catch the `IOException` first.